SmartDeviceLink/SDLRemoteControlCapabilities.h

```diff
@@ -0,0 +1,42 @@
+//
+//  SDLRemoteControlCapabilities.h
+//
+
+#import "SDLRPCMessage.h"
+@class  SDLClimateControlCapabilities;
+@class  SDLRadioControlCapabilities;
+@class  SDLButtonCapabilities;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLRemoteControlCapabilities : SDLRPCStruct
+
+-(instancetype) initWithClimateControlCapabilities:(NSArray<SDLClimateControlCapabilities *> *)climateControlCapabilities radioControlCapabilities:(NSArray<SDLRadioControlCapabilities *> *)radioControlCapabilities buttonCapabilities:(NSArray *)buttonCapabilities;
```

Comment:
needs nullability on the parameter types

Why no generic on the button capability type?

formatting